test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -138,13 +147,26 @@ public FunctionsSyncManagerTests()
 
             _hostNameProvider = new HostNameProvider(_mockEnvironment.Object);
 
+            var (serviceProvider, services) = CreateDefaultConfigurationWithHostJsonFileAndEnvBuilder();
```

Comment:
Once you're taking the new provider types as ctor parameters on FunctionsSyncManager, you shouldn't need to build a separate host like this

Reply:
I'll do some experiment because of your advice.

Reply:
It won't get it from Constructor. https://github.com/Azure/azure-functions-host/pull/7863#discussion_r794094949